docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -215,13 +215,20 @@ For details on the architecture design, see [Automatic resharding of data with t
 
 #### Enable automatic tablet splitting
 
-To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable-automatic-tablet-splitting) flag and specify the associated flags to configure when tablets should split, and use `yb-tserver` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-tserver/#enable-automatic-tablet-splitting). The flag usage must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one tablet per node by default.
 
-When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one shard **per _yb-tserver_** by default; from version `2.14.10` such tables have one shard (for servers with up to 2 CPU cores) or two shards (for servers with up to 4 CPU cores) **per _cluster_** by default.
+The following additional defaults apply:
+
+* From version 2.14.10, for servers with up to 2 CPU cores, newly-created tables have one tablet or, for servers with up to 4 CPU cores, two tablets **per _cluster_** by default.
+* From version 2.18, for servers with up to 2 CPU cores, newly-created tables have one tablet per table, and servers with up to 4 CPU cores have 2 tablets per table.
+
+From version 2.18.0, automatic tablet splitting is turned on by default.
+
+To control automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable-automatic-tablet-splitting) flag and specify the associated flags to configure when tablets should split, and use `yb-tserver` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-tserver/#enable-automatic-tablet-splitting). The flags must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
 
 {{< note title="Note" >}}
 
-Newly-created tables with [range sharding](../../../architecture/docdb-sharding/sharding/#range-sharding) always have one shard *per cluster* unless table partitioning is specified explicitly during table creation.
+Newly-created tables with [range sharding](../../../architecture/docdb-sharding/sharding/#range-sharding) always have one shard _per cluster_ unless table partitioning is specified explicitly during table creation.
```

Comment:
Maybe `one shard` should be replaced with `one tablet` to follow the style which is being used above in the changes.

Reply:
Also, sometimes it is 'per cluster' other times it is 'per table' - but (from previous comment) these appear to be equivalent, so will use 'per table'